### DISCUSSION TITLE
Check production befor payout

### DISCUSSION THREAD

**Discussion by dracou** [Lolliedieb/lolMiner-releases#1715] [Q&A]
Hello world!

I'm testing and using LOLminer for a week now and am mining ETC.
Everything seems to work fine with f2pool.

I understood that the payout is granted by 2 conditions: mining minimum 0.1ETC and waiting for the daily moment of payout (don't remember the time, but is NA wrt my question).

Question is: how can I know how many ETC I mined with LOLminer? Is there any command or something else that I can type on **windows** in order to check how far I am of my 0.1 ETC?

Thanks,

**Comment by esippert** [comment]
If you go to fpool, and click the little magnifying glass in the upper right corner, you can paste in your wallet address, and it'll bless you with stats, progress, payouts, etc... etc...
